### PR TITLE
Youtube and Twitch videos now provide more accurate times than before

### PR DESCRIPTION
**Pull request by jasonrsadler**
Addresses brave/brave-browser#971

See brave/brave-core#522 for test plan

### CODE REVIEW

File: include/bat/ledger/publisher_info.h [hunk 1]
```diff
@@ -51,9 +51,8 @@ LEDGER_EXPORT struct PublisherInfoFilter {
   int category;
   PUBLISHER_MONTH month;
   int year;
-  PUBLISHER_EXCLUDE excluded;
-
   std::vector<std::pair<std::string, bool>> order_by;
+  PUBLISHER_EXCLUDE excluded;
```
Comment (ryanml) [inline include/bat/ledger/publisher_info.h]: Can we flip this back? per: https://github.com/brave-intl/bat-native-ledger/pull/119

File: src/bat_get_media.cc [hunk 2]
```diff
@@ -118,12 +113,14 @@ void BatGetMedia::getPublisherInfoDataCallback(const std::string& mediaId, const
           _2,
           _3));
     } else if (TWITCH_MEDIA_TYPE == providerName) {
-      const std::string mediaUrl = getMediaURL(new_media_id, providerName);
+      const std::string twitchMediaID = 
+        braveledger_bat_helper::split(mediaId, '_')[0];
```
Comment (ryanml) [inline src/bat_get_media.cc]: Should we do a substring check for `_`? Just so we aren't accessing an index that doesn't exist. Let me know if `braveledger_bat_helper::split` accounts for this

Comment (jasonrsadler) [reply src/bat_get_media.cc]: Done.

File: include/bat/ledger/publisher_info.h (hunk 1 above)
Comment (jasonrsadler) [reply include/bat/ledger/publisher_info.h]: Done.

**Pull request merged**